.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,25 @@
 ## FreeCodeCamp Pull Request template
-Please, go through steps before you submit a PR.
+Please, go through these steps before you submit a PR.
 
 1. Make sure that your PR is not a duplicate.
-2. If not then:
+2. If not, then make sure that:
 
-  2.1. Make sure you've done your changes in a separate branch, like `fix/signin-issue` or `feature/issue-templates`.
+  2.1. You have done your changes in a separate branch. Branches MUST have descriptive names that start with either the `fix/` or `feature/` prefixes. Good examples are: `fix/signin-issue` or `feature/issue-templates`.
 
-  2.2. You have a descriptive commit message.
+  2.2. You have a descriptive commit message. The first line of the commit message is the commit title - please remember to keep it short. Use more lines to provide more details.
```

Comment:
Not sure if we need this here. We need to keep the template as short as possible.